### PR TITLE
Add deepseek LLMs at OpenAI plugin

### PR DESCRIPTION
**Pull request by lenage**
This MR does the following changes:
1.  Added support for DeepSeek LLM:
2. Updated examples: Modified minimal_assistant.py to use DeepSeek LLM
3. Refactored simple-color/agent.py to generate random colors instead of using a fixed color
4. Minor improvements:
   1) Added environment variable loading in simple-color/agent.py
   2) Updated WorkerOptions in simple-color/agent.py to include an agent name

### CODE REVIEW

File: examples/voice-assistant/minimal_assistant.py [hunk 1]
```diff
@@ -23,10 +23,11 @@ async def entrypoint(ctx: JobContext):
     assistant = VoiceAssistant(
         vad=silero.VAD.load(),
         stt=deepgram.STT(),
-        llm=openai.LLM(),
+        llm=openai.LLM.with_deepseek(),
```
Comment (theomonnom) [inline examples/voice-assistant/minimal_assistant.py:26]: Let's keep the default to openai for the minimal_assistant :)

File: examples/simple-color/agent.py [hunk 2]
```diff
@@ -25,13 +28,22 @@ async def _draw_color():
...
+            frame = rtc.VideoFrame(
+                WIDTH, HEIGHT, rtc.VideoBufferType.RGBA, argb_frame)
             source.capture_frame(frame)
 
     await _draw_color()
 
 
 if __name__ == "__main__":
-    cli.run_app(WorkerOptions(entrypoint_fnc=entrypoint))
+    cli.run_app(WorkerOptions(entrypoint_fnc=entrypoint, agent_name="simple-color"))
```
Comment (theomonnom) [inline examples/simple-color/agent.py:49]: This is awesome! definitely less confusing!
Can you remove the agent_name inside the WorkerOptions, adding agent_name currently requires manual dispatch of agents and this part isn't well documented yet

**Review by theomonnom — APPROVED** [review]
Thanks!!

File: examples/voice-assistant/minimal_assistant.py (hunk 1 above)
Comment (lenage) [reply examples/voice-assistant/minimal_assistant.py:26]: sure, just add as example, will remove in next commit

File: examples/simple-color/agent.py (hunk 2 above)
Comment (lenage) [reply examples/simple-color/agent.py]: sure, just removed at [43dcc62](https://github.com/livekit/agents/pull/714/commits/43dcc6285b52fc2bee61f837d4b0fca5b59f70e4)

**Comment by theomonnom** [comment]
https://github.com/byteb/agents/pull/1

**Comment by theomonnom** [comment]
🚀

**Pull request merged**